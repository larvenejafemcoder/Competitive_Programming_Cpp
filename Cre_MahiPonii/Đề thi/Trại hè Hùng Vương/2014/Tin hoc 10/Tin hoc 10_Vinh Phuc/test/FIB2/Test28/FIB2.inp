44
37 9472989
31 881991
37 9419480
6 8
20 5902
25 5620
12 203
26 14528
0 1
22 9744
13 222
10 26
45 1446740503
38 20183930
0 1
7 1
28 318797
2 2
38 35468465
12 214
34 5597974
27 211714
8 22
24 73582
40 43661414
28 306796
45 745202021
22 26196
9 14
17 964
26 184134
18 191
21 13362
30 535259
26 183919
19 746
21 15504
7 8
9 11
22 20891
27 103047
14 432
17 368
6 9
