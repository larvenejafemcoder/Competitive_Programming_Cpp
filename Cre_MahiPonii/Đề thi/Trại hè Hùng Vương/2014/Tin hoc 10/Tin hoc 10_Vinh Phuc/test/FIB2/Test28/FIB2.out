3618360
336891
3597921
3
2254
2147
78
5549
1
3722
85
10
552605699
7709575
1
0
121770
1
13547748
82
2138236
80868
9
28106
16677176
117186
284641844
10006
5
368
70333
73
5104
204451
70251
285
5922
3
4
7980
39361
165
141
4
